From aa485c8c876d36ac6a3255d2b7836757bd164e5c Mon Sep 17 00:00:00 2001
From: Yuwei Chen <yuwei.chen@intel.com>
Date: Wed, 29 Jul 2020 11:59:50 +0800
Subject: [PATCH] Test case9 for delete library from dsc component

Description: Delete the library (LockBoxLib|MdeModulePkg\Library\LockBoxNullLib\LockBoxNullLib.inf) from dsc component, which also write in Library class.

Repo: edk2
Signed-off-by: Yuwei Chen <yuwei.chen@intel.com>
---
 TestPkg/TestPkg32X64.dsc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/TestPkg/TestPkg32X64.dsc b/TestPkg/TestPkg32X64.dsc
index b24f19fc5f..22cbbb26ca 100644
--- a/TestPkg/TestPkg32X64.dsc
+++ b/TestPkg/TestPkg32X64.dsc
@@ -117,7 +117,7 @@
   TestPkg/Sec/SecMain.inf {
     <LibraryClasses>
     NULL|MdeModulePkg/Library/LzmaCustomDecompressLib/LzmaCustomDecompressLib.inf
-	LockBoxLib|MdeModulePkg\Library\LockBoxNullLib\LockBoxNullLib.inf
+	#LockBoxLib|MdeModulePkg\Library\LockBoxNullLib\LockBoxNullLib.inf
   }
 
   #
-- 
2.27.0.windows.1

